### ISSUE TITLE
Allow to set base dir

### ISSUE THREAD

**Issue by Suor** [xi/filecachetools#1]
Preferably support using several caches with different settings:

``` python
cache1 = Cache(path1, maxsize)
cache2 = LRUCache(path2, maxsize, ttl=ttl)

@cached(cache1)
def some_func(...):
    ...

@cached(cache2)
def other_func(...):
    ...
```

**Comment by xi** [comment]
As this is a simple change I implemented it in 8fc5700 and published it in version 0.3.0. I added it as a new keyword argument:

```
cache1 = Cache(name1, maxsize, base_dir=base_dir1)
cache2 = LRUCache(name2, maxsize, ttl=ttl, base_dir=base_dir2)
```

I am interested in the exact usecase though. Why is using a different name not enough?

**Comment by Suor** [comment]
Thanks, I can now remove my hack.

2015-11-10 3:11 GMT+07:00 Tobias Bengfort notifications@github.com:
